netty/src/main/java/io/grpc/netty/HandlerSettings.java

```diff
@@ -36,7 +36,7 @@
 
 /**
  * Allows autoFlowControl to be turned on and off from interop testing and flow control windows to
- * be accessed.
+ * be accessed. For internal testing only.
  */
 @VisibleForTesting
```

Comment:
I think the message should be included in the generated doc to make it clear to users.